### ISSUE TITLE
gen_data.py assumes that sql table exist, no code to create them?

### ISSUE THREAD

**Issue by ANogin** [dselsam/neurocore-public#2]
It looks like `python/gen_data.py` is assuming that a number of tables exist in the SQL database - `events`, `gd_runs` and there is no code to create them, and one is left trying to guess what the schema for those tables is supposed to be and then creating them manually.

**Comment by dselsam** [comment]
I tracked down my original schema file and made a gist: https://gist.github.com/dselsam/cd0f46c3ca366120c6074bf73d49a1f9

**Comment by ANogin** [comment]
Thank you! Will try it now. But as I was trying to recreate it manually, I realized that DB initialization seems to include not only the table schemas, but some actual preloading of SAT problems into the DB - so I guess would still need to figure that one out...

**Comment by dselsam** [comment]
Note: the problems take up a ton of space.

**Comment by ANogin** [comment]
Yes, I know. But do you have a script to populate the sql db you can share? Thanks!

**Comment by dselsam** [comment]
No. If I recall, many of the SATCOMP downloads were unreliable back then and I ended up retrying a lot manually at different times.

**Comment by ANogin** [comment]
The question is more on what exactly needs to go into the db once they are downloaded. I can mostly guess it from the schema, and can probably recreate the script for inserting them, but would obviously rather just use yours if you had one. There is also some references to `bname NOT LIKE 'randkcnf%%'` and `problem_id NOT IN (select problem_id from satcomp_info where year = 2018)`, so I am also wondering what else needs to be in DB besides SATCOMP 2018...

**Comment by dselsam** [comment]
Which SAT problems should be in the DB depends on what you are trying to do. Are you trying to train something for a more recent (or future) satcomp? Are you trying to train something for a particular application/problem-distribution? The repo was special-cased for a few specific experiments.

**Comment by ANogin** [comment]
The hope was to first reproduce what you did, as a way of making sure I know how to use your code, before trying anything else.
